onnx/helper.py

```diff
@@ -814,8 +812,7 @@ def str_list(str_elem: Callable[[_T], str], xs: Sequence[_T]) -> str:
             content.append("<Tensor>")
         else:
             # special case to print scalars
-            field = STORAGE_TENSOR_TYPE_TO_FIELD[attr.t.data_type]
-            content.append(f'<Scalar Tensor {str(getattr(attr.t, field))}>')
```

Comment:
It seems incorrectly used... We shouldn't use STORAGE_TENSOR_TYPE_TO_FIELD for a TensorProto so I just removed it.